datapreview.tex

```diff
@@ -58,23 +61,24 @@ \subsubsection{Data Preview 0.2}
-Table~\ref{tab:dp-zero-three-products} presents a summary of the expected DP0.3 data products.
-The exact data products for DP0.3 are still to be decided. 
+Delivered in June  2023, DP0.3 is the last in the DP0 series of Data Previews based on simulated LSST-like data. 
+DP0.3 supports the Solar System Science Collaboration by hosting their simulated 10-year catalog and serving it via the RSP at the US DAC. 
+Table~\ref{tab:dp-zero-three-products} presents a summary of the  DP0.3 data products.
+
+For full details on DP0.3 including an exact description of the data products served, see the documentation at \url{https://dp0-3.lsst.io/}
 
 \subsection{Data Preview 1}
 \label{sec:dp1}
 
-Data Preview 1 was originally defined to be based on reprocessed on-sky data taken with ComCam.
-Following the replan of the Construction project in December 2022, no on-sky data will be taken with ComCam, \S~\ref{sec:commissioning}.
-Consequently DP1 has been redefined to be based on a subset of science-grade images taken with LSSTCam during a period of a few days around the System First Light milestone, \S~\ref{ssec:commissioning-observations}.
-The processing pipelines and exact data products that will comprise DP1 are still under discussion. 
-At minimum, DP1 will deliver visit-level images and catalogs to enable initial studies of observational and instrumental effects. 
+Data Preview 1 (DP1) will provide the community with a small set of useful science-grade data as soon as possible following System First Light (\S~\ref{ssec:commissioning-milestones}) to enable them to understand the characteristics of and prepare to work with LSST data. 
+Depending on the commissioning schedule (\S~\ref{ssec:commissioning-schedule}) and  performance of the integrated system at the time, Data Preview 1 will be based either on ComCam  or early LSSTCam on-sky data. 
+Exactly which scenario plays out will not be known until LSSTCam is ready to ship to the summit. 
 
-Note that the DP1 period of time during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken in the few days around System First Light.
-Table~\ref{tab:dp-one-products} presents a summary of the data products expected in DP1.
+In both DP1 scenarios, the exact processing pipelines and data products are still under discussion.
+At minimum, both DP1 scenarios  will deliver visit-level images and catalogs to enable initial studies of observational and instrumental effects. 
+Table~\ref{tab:dp-one-products} presents a current best idea of  the data products that might  comprise DP1. 
 
+Note that the time period during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken around System First Light.
```

Comment:
"Note that the time period during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken by the end of ComCam work on sky and/or around System First Light."